manifests/g/GnuWin32/Flex/2.5.4a-1/GnuWin32.Flex.installer.yaml

```diff
@@ -0,0 +1,12 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: GnuWin32.Flex
+PackageVersion: 2.5.4a-1
+ReleaseDate: 2004-04-07
+InstallerType: inno
+Installers:
+- InstallerUrl: https://kumisystems.dl.sourceforge.net/project/gnuwin32/flex/2.5.4a-1/flex-2.5.4a-1.exe
```

Comment:
Please use the generic sourceforge download link and not a specific CDN

Reply:
I remove kumisystems and this link does not work, any idea?

@Trenly

Reply:
> I remove kumisystems and this link does not work, any idea?
> 
> @Trenly

Because just removing the CDN endpoint doesn’t get you the generic URL.

You need to use https://downloads.sourceforge.net/project/gnuwin32/flex/2.5.4a-1/flex-2.5.4a-1.exe

Reply:
Thanks, I corrected the URL :)